### PR TITLE
Regex: Two more regex added to context menu + multi-lang support

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: LanguageMaster.xml [hunk 1]
```diff
@@ -1421,8 +1421,10 @@ can edit in same subtitle file (collaboration)</Information>
     <NewLine>New line (\r\n)</NewLine>
     <NewLineShort>New line (\n)</NewLineShort>
     <AnyDigit>Any digit (\d)</AnyDigit>
+    <NonDigit>Non digit (\D)</NonDigit>
```
Comment (xylographe) [inline LanguageMaster.xml]: I would suggest `<NonDigit>Not a digit (\D)</NonDigit>`.

Also, (line 1420)
 `<NonWordBoundary>Non word boundary (\B)</NonWordBoundary>`
 ==> `<NonWordBoundary>Not a word boundary (\B)</NonWordBoundary>`.

File: LanguageMaster.xml [hunk 2]
```diff
@@ -1421,8 +1421,10 @@ can edit in same subtitle file (collaboration)</Information>
...
     <AnyCharacter>Any character (.)</AnyCharacter>
     <AnyWhitespace>Any whitespace (\s)</AnyWhitespace>
+    <NonSpaceCharacter>Non space character (\S)</NonSpaceCharacter>
```
Comment (xylographe) [inline LanguageMaster.xml]: `<NonWhitespace>Non-whitespace (\S)</NonWhitepace>`

Comment (ivandrofly) [inline LanguageMaster.xml]: Note that \s won't only match ` ` \r\n are also included

Comment (xylographe) [inline LanguageMaster.xml]: Space is used for ASCII 32 (U+0020).
Whitespace[1] is used for a _set_ of characters, including space, nbsp, tab, linefeed, and others.
`\s` represents the set of all white space characters, `\S` is its complement (the set of characters that are not in `\s`).

[1] Actually, two words "white space", but "whitespace" seems to be more commonly used these days.

Comment (ivandrofly) [inline LanguageMaster.xml]: Well, let's hear also what @niksedk will say about it :confused:

Comment (niksedk) [inline LanguageMaster.xml]: English is not my native language, but "non-digit character" / "non-whitespace character" looks good to me - like from http://www.w3schools.com/jsref/jsref_regexp_digit_non.asp

**Pull request merged**